package.json

```diff
@@ -225,10 +225,10 @@
         "properties-parser": "^0.3.1",
         "query-string": "5.1.1",
         "raf": "^3.4.0",
-        "react": "^16.8.0",
+        "react": "16.9.0",
```

Comment:
Can we leave the `^` here or is the lockdown intentional?

Reply:
leaving it